lib/generators/templates/spec.erb

```diff
@@ -0,0 +1,21 @@
+require 'spec_helper'
+
+RSpec.describe <%= class_name %> do
+  describe 'call' do
```

Comment:
Agreed in general, but there are still several ways and styles of testing that one public method call.  While I agree that the code template here is a good example of how to write an interactor spec, I don't think it's a good skeleton.

That said, it's not a huge deal.  As you said, it's quick enough to delete those lines if you don't want them.

Also, thanks for pulling this together.  :+1: